app/Commands/Session/EndCommand.php

```
@@ -0,0 +1,174 @@
+                $lines[] = "Commits: {$commitCount}";
+                $lines[] = '';
+                $lines[] = 'Recent commits:';
+                foreach (array_slice(array_filter(explode("\n", trim($commits))), 0, 5) as $commit) {
+                    $lines[] = "- {$commit}";
+                }
+            }
+        }
+
+        // Get modified files
+        $modified = shell_exec('git diff --name-only HEAD~1 2>/dev/null');
+        if ($modified !== null && trim($modified) !== '') {
```

PHPStan (2.1.33)

Parameter #1 $string of function trim expects string, string|false given.


(argument.type)